var/ramble/repos/builtin/applications/nvidia-hpl/application.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2022-2024 The Ramble Authors
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+from ramble.appkit import *
+
+from ramble.base_app.builtin.hpl import Hpl as HplBase
+
+
+class NvidiaHpl(HplBase):
+    """This application defines how to run NVIDIA's optimized version of HPL,
+    which is contained in NVIDIA's HPC-Benchmarks collection.
+
+    The NVIDIA HPC-Benchmarks collection provides four benchmarks (HPL,
+    HPL-MxP, HPCG, and STREAM) widely used in the HPC community optimized for
+    performance on NVIDIA accelerated HPC systems.
+
+    NVIDIA's HPL and HPL-MxP benchmarks provide software packages to solve a
+    (random) dense linear system in double precision (64-bit) arithmetic and in
+    mixed precision arithmetic using Tensor Cores, respectively, on
+    distributed-memory computers equipped with NVIDIA GPUs, based on the Netlib HPL
+    benchmark and HPL-MxP benchmark.
+
+    NVIDIA's HPCG benchmark accelerates the High Performance Conjugate Gradients
```

Comment:
Should we exclude the descriptions for HPCG and STREAM?

Reply:
👍🏻  Done